### ISSUE TITLE
Icon request: icon-meds-pills 

### ISSUE THREAD

**Issue by concordian** [FortAwesome/Font-Awesome#1778]
Category: medical

icon-name:  icon-meds-pills

graphic:  2-3 loose pills (not in pill box or bill bottle)

My graphic skills aren't sufficient to create a draft.

**Comment by RVXD** [comment]
+1

**Comment by kino90** [comment]
+1

**Comment by alxfoster** [comment]
+1
Just a simple pill would be fantastic:
Eg:
![pill](https://cloud.githubusercontent.com/assets/11067158/6258540/391348e0-b77e-11e4-94f5-994da52d596c.png)

**Comment by phpforfree** [comment]
+1

**Comment by bluebit27** [comment]
+1

**Comment by Dret42** [comment]
+1

**Comment by kino90** [comment]
I've done a little 'hack' on my css file to ad a custom `.fa-pill` using `.fa-toggle-on` and a 45deg rotation.. 

Here's the code:
```css
.fa-pill{
  filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=.5);
  -webkit-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
}
.fa-pill:before {
  content: "\f205";
}	
```

It's not good as having a real icon, but it does look good!

![pill](https://cloud.githubusercontent.com/assets/1820036/7453281/d60fe616-f267-11e4-8cbf-53e688d04ab8.PNG)

**Comment by ringner** [comment]
+1

**Comment by AustrianDudes** [comment]
+1

**Comment by jeffgarretson** [comment]
+1

**Comment by jo-han** [comment]
+1

**Comment by cguzman1028** [comment]
+1

**Comment by eddieebeling** [comment]
+1

**Comment by Heinerion** [comment]
+1

**Comment by krishatcher** [comment]
+1

**Comment by G-Driver** [comment]
+1

**Comment by lesula** [comment]
+1

**Comment by nxtdesigns** [comment]
+1

**Comment by chmoder** [comment]
+1

**Comment by jldavis33** [comment]
+1 — My idea, similar to others. 
![pill](https://cloud.githubusercontent.com/assets/5025107/12523056/f1248c50-c119-11e5-8c3a-875bae3d1d05.png)

**Comment by lukeemery** [comment]
+1

**Comment by pitus** [comment]
+1 for Daily Log - to quickly enter when a Pill was taken by clicking on the icon

**Comment by dtraverso** [comment]
+1

**Comment by frankdl98** [comment]
+1

**Comment by ActionableResearch** [comment]
+1

**Comment by dscaglione** [comment]
+1

**Comment by runderworld** [comment]
+1

**Comment by pl21** [comment]
+1

**Comment by MatthijsMali** [comment]
+1

**Comment by mrjlynch** [comment]
+1

**Comment by cema93** [comment]
+1

**Comment by shonezlo** [comment]
+1 need it for a CMS that deals with vitamins

**Comment by dmytro-savochkin** [comment]
+1

**Comment by cozzbie** [comment]
+1 Its shocking that since 2013 when this was first logged nothing has been done. Wow!! Please we need a pill icon.

**Comment by G-Driver** [comment]
+1

**Comment by kyurkchyan** [comment]
+1

**Comment by dripoll24** [comment]
+1

**Comment by tavomoya** [comment]
+1

**Comment by haoleboy** [comment]
+1

**Comment by srkela** [comment]
+1

**Comment by MatthijsMali** [comment]
@tagliala what is needed to make this happen? Apart from a lot of +1's?

**Comment by cozzbie** [comment]
Even +1s don't seem to be working at this point. Its been over 3 years of +1s @MatthijsMali @tagliala

**Comment by tagliala** [comment]
+1s work. [Ask them](https://github.com/FortAwesome/Font-Awesome/issues?q=is%3Aissue+is%3Aclosed+sort%3Acomments-desc) 😁 

If you need this icon in a short while, [customize Font Awesome](https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome)

**Comment by vaillancourtmax** [comment]
Here's another +1. :)

**Comment by cozzbie** [comment]
+1
+1
+1
+1
+1
+1
+1
+1
+1
+1
+1
+1
+1
+1
+1

**Comment by a2net** [comment]
+1

**Comment by MUVEN921** [comment]
+1

**Comment by cozzbie** [comment]
+20

**Comment by wesly--** [comment]
+1

**Comment by TheInfestation** [comment]
+1

**Comment by kimberley-johnston** [comment]
+1

**Comment by cozzbie** [comment]
Totally lost hope that this will ever happen.

**Comment by corlen** [comment]
+1

**Comment by alexnum** [comment]
+1

**Comment by andreschabur** [comment]
+1

**Comment by andre1810** [comment]
+1

**Comment by sensibleworld** [comment]
This is coming in the next release!

**Comment by sensibleworld** [comment]
This icon is included in the latest 5.0.7 release of Font Awesome! The solid style is in Font Awesome 5 Free, with the regular and light styles included in Font Awesome 5 Pro. https://fontawesome.com/icons?d=gallery&q=pills
